src/main/java/com/store/book/security/Authentication.java

```diff
@@ -0,0 +1,7 @@
+package com.store.book.security;
+
+public interface Authentication {
+    Object getPrincipal();
+
+    Object getCredentials();
```

Comment:
Why do you need it?

Reply:
This interface was initially planned for custom security logic, but currently it's not used and can be removed.